src/parser/core/tests/snapshotTest.js

```diff
@@ -55,9 +55,10 @@ export function tab(analyzer, parser=null) {
  */
 export default function snapshotTest(parserClass, moduleClass, key, propFn = statistic) {
   return () => {
-    const log = loadLogSync(key);
-    const parser = parseLog(parserClass, log);
-    expectSnapshot(parser, moduleClass, propFn);
+    return loadLog(key).then(log => {
+      const parser = parseLog(parserClass, log);
```

Comment:
Does this parse the log for every single module now?

Reply:
no, `integrationTest` does not use `snapshotTest` internally (it uses `expectSnapshot`).

However, I'm not sure if `snapshotTest` as-is should live on. If people start using it a lot then it will slow down test times by constantly re-parsing logs. Might be better to just provide a helper to do `beforeAll` parsing and then tell people to use `expectSnapshot`.